makefile

```
@@ -8,6 +8,9 @@ install:
 	@sudo cp ./dxhd /usr/bin/dxhd
 	@sudo mkdir -p /usr/share/licenses/dxhd/
 	@sudo cp LICENSE /usr/share/licenses/dxhd/
+	@test -d /usr/lib/systemd/user \
+		&& sudo cp systemd/dxhd.service /usr/lib/systemd/user/ \
+		||echo Systemd not found and user service unit not installed!
```

```suggestion
		&& sudo cp systemd/dxhd.service /usr/lib/systemd/user/ 
```

let's get rid of this message, no need to inform users of other init systems that.

---

If you remove `echo` with the error message, then `make` will of course crash with an installation error, because `test` exited with exit code 1. May be leave echo with empty mesage?

---

ah got you. in that case, do:
```suggestion
		&& sudo cp systemd/dxhd.service /usr/lib/systemd/user/ \
		|| true
```

---

Yes! Or so. :-)